src/app/zap-templates/zcl/data-model/chip/administrator-commissioning-cluster.xml

```diff
@@ -18,6 +18,7 @@ limitations under the License.
   <domain name="General"/>
 
   <enum name="StatusCode" type="ENUM8">
+    <cluster code="0x003c"/>
```

Comment:
This would not be needed if the enum was "scoped" under the Cluster below (i.e. `<enum>` tag moved inside `<cluster>` tag). Why is this very "non-XML" method used when proper XML hierarchy would make this easier ?

Reply:
I think it is for possible future use, one enum type is possibly used by multiple cluster in the future.

Reply:
we would move enum and struct inside cluster when https://github.com/project-chip/zap/issues/234 is resolved. we would move forward and unblock the complicated type code generation work for issue 8895

Reply:
I agree that this isn't how you'd write it in XML. I'm concerned that cluster authors will either forget to add that tag, or get that tag value wrong.